src/Uno.UI/UI/Xaml/Internal/PointerCapture.cs

```diff
@@ -0,0 +1,346 @@
+﻿using System;
```

Comment:
`#nullable enable`

Reply:
(Fix coming ... and found 3 places in the class of a possible nuff ref 😨)